File: Modul 2/Bahasa Python (Modul 2)/PRAK205-2310817320015-AlfisyahRinaFajriati.py
#case1
print("Input")
a,b = map(float, input().split())

alas = (b**2 - a**2) **(0.5)
tinggi = a
keliling = a+b+alas
luas = 1/2*alas*a

print("\nOutput")
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

#case2
print("\nInput")
a = float(input(" "))
b = float(input(" "))

alas = (b**2 - a**2) **(0.5)
tinggi = a
keliling = a+b+alas
luas = 1/2*alas*tinggi

print("\nOutput")
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul 2/Bahasa C (Modul 2)/PRAK205-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
#include <math.h>
int main(){
    printf("Input\n");
    float a,b;
    printf("");
    scanf("%f" "%f" , &a, &b);

    float alas = sqrt(b*b - a*a);
    float tinggi = a;
    float keliling = a+b+alas;
    float luas = 0.5*alas*a;

    printf("\nOutput\n");
    printf("Alas : %.0f\n", alas);
    printf("Tinggi : %.0f\n", tinggi);
    printf("Keliling : %.0f\n", keliling);
    printf("Luas : %.0f cm^2\n", luas);

    return 0;
}